doc/development-environment.md

```diff
@@ -208,6 +208,16 @@ Set the CMake path:
     "cmake.cmakePath": "path/to/cmake.exe"
 ```
 
+Set toolchain prefix in .vscode/settings.json:
+```
+"TOOLCHAIN_PREFIX":"c:/Program Files (x86)/GNU Arm Embedded Toolchain/9 2020-q2-update/",
```

Comment:
Please change the line to
`"TOOLCHAIN_PREFIX":"/path/to/toolchain",`

The path for the toolchain may not be the one you are proposing. Also a different version of the toolchain will produce a different path.

Reply:
It looks like that the original repository doesn't exist anymore.

Also, after reflection we think that this addition is not necessary.

At `Usage` step  2 we indicate that one can edit `.vscode/settings.json` file.
When editing this file one can notice that examples for Windows, Linux and MacOS are provided for `TOOLCHAIN_PREFIX` and `OPENOCD_BIN`

```json
// Place your settings in this file to overwrite default and user settings.
{
    "cmake.configureSettings": {

        // In case your GNU ARM-Toolchain is not installed under the default
        // path:
        //     Windows : No default path. Specify the path where the
        //               toolchain is installed. i.e:
        //               "C:/PROGRA~2/GNUTOO~1/92019-~1".
        //     Linux   : /usr
        //     OSX     : /usr/local
        // It is required to uncomment and to fill the following line.
        //"TOOLCHAIN_PREFIX":"/path/to/toolchain",

        // In case your OpenOCD is not installed under the default path:
        //     Windows : C:/openocd/bin/openocd.exe
        //     Linux   : /usr/bin/openocd
        //     OSX     : /usr/local/bin/openocd
        // Please uncomment the following line and fill it accordingly.
        //"OPENOCD_BIN":"C:/openocd/bin/openocd.exe",

...
}
```

If you don't mind we close this pull-request.